--- src/resources/image.h.orig	2006-10-29 23:17:41.000000000 +0100
+++ src/resources/image.h	2006-10-30 15:59:28.705199088 +0100
@@ -24,7 +24,7 @@
 #ifndef _TMW_IMAGE_H
 #define _TMW_IMAGE_H
 
-#include <SDL.h>
+#include <SDL/SDL.h>
 #ifdef USE_OPENGL
 
 /* The definition of OpenGL extensions by SDL is giving problems with recent
@@ -33,7 +33,7 @@
  */
 #define NO_SDL_GLEXT
 
-#include <SDL_opengl.h>
+#include <SDL/SDL_opengl.h>
 #endif
 
 #include "resource.h"
