--- tmw-0.0.21/tmw.desktop~	2006-09-17 23:02:37.000000000 +0200
+++ tmw-0.0.21/tmw.desktop	2006-10-24 17:42:02.000000000 +0200
@@ -8,9 +8,11 @@
 Comment[de]=ein Online Fantasy Spiel
 Comment[it]=Un gioco fantasy online
 Comment[nl]=Een online role playing game
+Comment[pl]=Internetowa gra fantasy RPG
 Exec=tmw
 StartupNotify=false
 Terminal=false
 Type=Application
 Icon=tmw.png
-Categories=Application;Game
+Categories=Game;RolePlaying;
+# vi: encoding=utf-8
